__tests__/app.test.js

```diff
@@ -106,23 +106,103 @@ describe('/GET', () => {
                 });
             })
         })
-        test('GET: 200 Return an empty array if given a valid article id with no comments', async () => {
-            const response = await request(app).get('/api/articles/8/comments');
+        test('GET: 200 Return an empty array if given a valid article ID with no comments', async () => {
+            const { status, body: { comments } } = await request(app).get('/api/articles/8/comments');
 
-            expect(response.status).toBe(200);
-            expect(response.body.comments.length).toBe(0);
+            expect(status).toBe(200);
+            expect(comments.length).toBe(0);
         })
         test('GET: 400 Return an error if given an invalid article id', async () => {
-            const response = await request(app).get('/api/articles/abc/comments');
+            const { status, body } = await request(app).get('/api/articles/abc/comments');
 
-            expect(response.status).toBe(400);
-            expect(response.body).toEqual({msg: 'Bad Request'});
+            expect(status).toBe(400);
+            expect(body).toEqual({msg: 'Bad Request'});
         })
         test('GET: 404 Return an error if given a non-existent article id', async () => {
-            const response = await request(app).get('/api/articles/99999/comments');
+            const { status, body } = await request(app).get('/api/articles/99999/comments');
+
+            expect(status).toBe(404);
+            expect(body).toEqual({msg: `No article was found with the id 99999`});
+        })
+    })
+})
+
+describe('/POST', () => {
+    describe('Comments', () => {
+        test('POST: 201 Adds a comment to the database and returns the posted comment', async () => {
+
+            const commentToPost = {
+                username: 'butter_bridge',
+                body: 'Hello World!'
+            }
+
+            const { status, body: { comment } } = await request(app).post('/api/articles/1/comments').send(commentToPost);
+
+            expect(status).toBe(201);
+            expect(comment).toMatchObject({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: commentToPost.username,
+                body: commentToPost.body,
+                article_id: expect.any(Number),
+            });
+        })
+        test('POST: 400 Return an error if given an invalid comment format', async () => {
+            const commentToPost = {
+                user: 'butter_bridge',
+                text: 'Hello World!'
+            }
+
+            const { status, body } = await request(app).post('/api/articles/1/comments').send(commentToPost);
+
+            expect(status).toBe(400);
+            expect(body).toEqual({msg: 'Bad Request'});
+        })
+        test('POST: 400 Returns an error if post has additional keys', async () => {
```

Comment:
This should be a 201 as it still has the 2 required properties needed to make a successful post 👍